<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>742</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="homeTab">
       <attribute name="title">
        <string>Home Apps</string>
       </attribute>
       <widget class="QLabel" name="path_header">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>551</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Source:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 0, 0);
font: 12pt &quot;MS Serif&quot;;</string>
        </property>
        <property name="text">
         <string>Desktop Links</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>50</y>
          <width>141</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Web Links</string>
        </property>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>330</y>
          <width>591</width>
          <height>21</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>350</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 127);
font: 12pt &quot;Microsoft YaHei&quot;;</string>
        </property>
        <property name="text">
         <string>Lists</string>
        </property>
       </widget>
       <widget class="QListWidget" name="desktop_list">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>281</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
       <widget class="QListWidget" name="web_list">
        <property name="geometry">
         <rect>
          <x>325</x>
          <y>100</y>
          <width>291</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
       <widget class="QListWidget" name="lists_list">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>380</y>
          <width>591</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="settings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>(Assets path)</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="assets_path">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>80</y>
          <width>381</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QToolButton" name="browse_assets">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>80</y>
          <width>27</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>290</y>
          <width>521</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>** Just restart the appilcation to get updated path</string>
        </property>
       </widget>
       <widget class="QPushButton" name="defBtn">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>160</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>To default</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>678</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd_refs">
    <property name="title">
     <string>Add App</string>
    </property>
    <addaction name="add_desktop"/>
    <addaction name="add_web"/>
    <addaction name="add_list"/>
   </widget>
   <addaction name="menuAdd_refs"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDesktop_Apps">
   <property name="text">
    <string>Desktop Apps</string>
   </property>
  </action>
  <action name="actionWeb_Apps">
   <property name="text">
    <string>Web Apps</string>
   </property>
  </action>
  <action name="actionNormal_Links">
   <property name="text">
    <string>Normal Links</string>
   </property>
  </action>
  <action name="add_desktop">
   <property name="text">
    <string>+ Desktop App</string>
   </property>
  </action>
  <action name="add_web">
   <property name="text">
    <string>+ Web App</string>
   </property>
  </action>
  <action name="add_list">
   <property name="text">
    <string>+ List</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
